### PR TITLE
moving dependecymodel and platformabstraction to preview1-001960

### PR DESCRIPTION
**Pull request by ramarag**
@eerhardt @livarcocc PTAL, this is a step towards https://github.com/dotnet/core-setup/issues/1840
and followup on https://github.com/dotnet/core-setup/pull/1986

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x86 Debug Build
@dotnet-bot Test Ubuntu16.10 x64 Debug Build
@dotnet-bot Test RHEL7.2 x64 Release Build

File: build_projects/dotnet-cli-build/dotnet-cli-build.csproj [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <!-- This dependency was added due to an issue in restore where a lower version of this package coming from nuget.commandline.xplat
     led to an error. This is tracked as NuGet issue : https://github.com/NuGet/Home/issues/4213 -->
     <PackageReference Include="Microsoft.Build.Framework" Version="$(CLI_MSBuild_Version)" />
-    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(PlatformAbstractionsVersion)" />
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="1.0.3" />
```
Comment (livarcocc) [inline build_projects/dotnet-cli-build/dotnet-cli-build.csproj]: Why is this necessary?

Comment (ramarag) [inline build_projects/dotnet-cli-build/dotnet-cli-build.csproj]: We are upgrading the cli to use 2.0 for platformabstractions but the bootstrapped cli depends on the 1.0.3

Once the new CLI comes out we can revert this change

**Comment by ramarag** [comment]
@livarcocc @eerhardt signoff ?

File: TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
-    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.12-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;ubuntu.16.10-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.24-x64;opensuse.42.1-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.12-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;ubuntu.16.10-x64;centos.7-x64;rhel.7-x64;debian.8-x64;fedora.24-x64;opensuse.42.1-x64</RuntimeIdentifiers>
```
Comment (eerhardt) [inline TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj]: I'm surprised this change is necessary.

Comment (ramarag) [inline TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj]: otherwise the default resore for rhel populates for rid rhel.7.2-x64,
while our tests work with rid rhel.7-x64

Comment (eerhardt) [inline TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj]: Ah, I see it now.  We've updated to the latest PlatformAbstractions, which the RID logic has been changed for RedHat.  This makes sense to me now.
